llvm/test/CodeGen/AArch64/machine-cp-sub-reg.mir

```diff
@@ -0,0 +1,33 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 4
+# RUN: llc -o - %s -O3 --run-pass=machine-cp -mcp-use-is-copy-instr -mtriple=arm64-apple-macos -mcpu=apple-m1 --verify-machineinstrs | FileCheck %s
```

Comment:
Can you remove `-O3` and maybe the -mcpu?

Reply:
Yes. I can delete them all.